.circleci/config.yml

```diff
@@ -25,50 +25,60 @@ jobs:
       - checkout
       - run: 
           name: Install client
-          command: cd client && yarn install --frozen-lockfile
+          command: cd client && npm install
```

Comment:
always `npm ci` for installing in the ci context. 

The difference is that `npm ci` requires a `package-lock.json` and installs the exact versions of all dependencies specified there, whereas `npm install` checks the allowed version ranges in `package.json` against what newer versions are available for each package and its recursive dependencies, and then updates `package-lock.json` to newer versions as allowed by the rules in `package.json`